### ISSUE TITLE
Restore from previous backup link does not work during onboarding.

### ISSUE THREAD

**Issue by kragoth** [home-assistant/frontend#10522]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I am unable to upload a previous backup when onboarding a new home assistant installation.

When clicking on the "Alternatively you can restore from a previous backup." link, nothing happens.

### Describe the behavior you expected

Expected outcome:
* A dialog should allow me to upload a backup file

### Steps to reproduce the issue

Scenario
1. Create a brand new home assistant installation
* I used the .vdi image from here: https://www.home-assistant.io/installation/linux#install-home-assistant-operating-system
2. Once the installation is ready navigate to: http://homeassistant.local:8123/
3. When presented with the "Create Account" or "Alternatively you can restore from a previous backup." click on the "Alternatively you can restore from a previous backup." link.

### What version of Home Assistant Core has the issue?

2021.11.0

### In which browser are you experiencing the issue with?

Google Chrome 95.0.4638.69

### Which operating system are you using to run this browser?

Windows 11

### Javascript errors shown in your browser console/inspector

```txt
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'localize')
    at HTMLElement.value (dialog-hassio-backup-upload.ts:57)
    at HTMLElement.update (styles.ts:365)
    at HTMLElement.performUpdate (styles.ts:365)
    at HTMLElement.scheduleUpdate (styles.ts:365)
    at HTMLElement._$EC (styles.ts:365)
```

### Additional information

I have tried in Edge, Firefox and Chrome. All browsers experience the same issue.

**Comment by Misiu** [comment]
I think this is duplicate of https://github.com/home-assistant/frontend/issues/10516

**Comment by ludeeus** [comment]
That it is :+1: 
Closing this as a duplicate, both will be solved with <https://github.com/home-assistant/frontend/pull/10523>
